R/polars_options.R

```diff
@@ -0,0 +1,84 @@
+#' Get and reset polars options
+#'
+#' @description
+#' `polars_options()` returns a list of options for polars. Options
+#' can be set with [`options()`]. Note that **options must be prefixed with
+#' "polars."**, e.g to modify the option `int64_conversion` you need to pass
+#' `options(polars.int64_conversion =)`. See below for a description of all
+#' options.
+#'
+#' `polars_options_reset()` brings all polars options back to their default
+#' value.
+#'
+#' @details The following options are available (in alphabetical order, with the
+#'   default value in parenthesis):
+#'
+#' * `int64_conversion` (`"double"`): How should Int64 values be handled when
+#'   converting a polars object to R?
+#'    * `"double"` converts the integer values to double.
+#'    * `"integer"` converts to the R's [integer] type. If the value is out of
+#'      the range of R's integer type, export as [NA_integer_].
+#'    * `"integer64"` uses `bit64::as.integer64()` to do the conversion
+#'      (requires the package `bit64` to be attached).
+#'    * `"character"` converts Int64 values to character.
+#' * `df_knitr_print` (TODO: possible values??)
```

Comment:
I don't see any usage of `df_knitr_print` in the package so far, so leaving this to be completed